cybersource.yml

```diff
@@ -8,13 +8,17 @@
 #    :username: <%= ENV['OD_USERNAME'] %>
 #    :password: <%= ENV['OD_PASSWORD'] %>
 #    :check_for_duplicates: true
+  :test: true
```

Comment:
`:test` is set to true by default. Also, it is already set on line 3 (per merchant account)?